README.md

```diff
@@ -219,7 +219,7 @@ OpenShift templates can be found [here](/openshift/qontract-reconcile.yaml). In
 
 ## Installation
 
-This project targets Python version 3.9.x for best compatibility. Verify the Python3 version that your shell is using with `python3 --version`. You can optionally use a tool like [pyenv](https://github.com/pyenv/pyenv) to manage Python versions on your computer.
+This project targets Python version 3.11.x for best compatibility. Verify the Python3 version that your shell is using with `python3 --version`. You can optionally use a tool like [pyenv](https://github.com/pyenv/pyenv) to manage Python versions on your computer.
```

Comment:
asdf was also mentioned elsewhere, do you want to add it here as an alternative to pyenv? And should we introduce in the repo file-based version selection (for both pyenv and asdf?)

Reply:
```
$ git grep '3\.9'
Makefile:       python3.9 -m venv venv
README.md:This project targets Python version 3.9.x for best compatibility. Verify the Python3 version that your shell is using with `python3 --version`. You can optionally use a tool like [pyenv](https://github.com/pyenv/pyenv) to manage Python versions on your computer.
reconcile/utils/terrascript_aws_client.py:            lambda_values["runtime"] = common_values.get("runtime", "python3.9")
setup.py:    python_requires=">=3.9",
setup.py:        "Programming Language :: Python :: 3.9",
```

Reply:
See [1f761c1](https://github.com/app-sre/qontract-reconcile/pull/3672/commits/1f761c193a23eb855255e3f804df0fbb00907ec3)